src/sql/base/browser/ui/breadcrumb/breadcrumb.component.ts

```diff
@@ -16,7 +16,7 @@ import { subscriptionToDisposable } from 'sql/base/browser/lifecycle';
 @Component({
 	selector: 'breadcrumb',
 	template: `
-				<span style="display: flex; flex-flow: row; align-items: center; margin: 10px">
+				<span style="display: flex; flex-flow: row; align-items: center; margin: 10px; margin-left: 19px">
```

Comment:
a good candidate for a css class.

Reply:
moved to new css class "breadcrumb-container"